DependencyInjection/CaxyHtmlDiffExtension.php

```diff
@@ -43,11 +43,14 @@ public function load(array $configs, ContainerBuilder $container)
 
         $cacheDriverId = null;
 
-
         if (!empty($config['doctrine_cache_driver']) && $config['doctrine_cache_driver']['enabled']) {
             $cacheDriverId = $this->loadCacheDriver('doctrine_cache_driver', $config['doctrine_cache_driver'], $container);
         }
 
+        if (!isset($config['purifier_cache_location']) {
+            $config['purifier_cache_location'] = $container->getParameter('kernel.cache_dir');
+        }
```

Comment:
We'll still need to add the method call so that it gets called on the HtmlDiffConfig service - you can simply add it to the `$methodsToCall` array in the `loadHtmlDiffConfig` function below - around line 73.

```php
$methodsToCall = array(
    // ...
    'purifier_cache_location' => 'setPurifierCacheLocation',
);
```